ruby/lib/minitest/queue/runner.rb

```diff
@@ -104,7 +158,7 @@ def queue_url
       end
 
       def abort!(message)
-        puts message
+        step(red(message), collapsed: false)
```

Comment:
Should this be it's own step, or should it uncollapse an existing one?

Reply:
good point.